Zeeguu Reader/Zeeguu Reader/HistoryItemViewController.swift

```diff
@@ -0,0 +1,137 @@
+//  Copyright © 2015 Jorrit Oosterhof.
+// 
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files (the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions:
+//  
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//  
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+//
+
+import UIKit
+import ZeeguuAPI
+
+class HistoryItemViewController: UIViewController {
+
+	let bookmark: Bookmark
+	
+	private let translationLabel = UILabel.autoLayoutCapapble()
+	private let languageLabel = UILabel.autoLayoutCapapble()
+	private let contextLabel = UILabel.autoLayoutCapapble()
+	
+	init(bookmark: Bookmark) {
+		self.bookmark = bookmark
+		super.init(nibName: nil, bundle: nil)
+	}
+
+	required init?(coder aDecoder: NSCoder) {
+	    fatalError("init(coder:) has not been implemented")
+	}
+	
+	override func viewDidLoad() {
+		super.viewDidLoad()
+		// Do any additional setup after loading the view, typically from a nib.
+		self.view.backgroundColor = UIColor.whiteColor()
+		
+		self.title = bookmark.word
+		
+		let sv = UIScrollView.autoLayoutCapapble()
+		let v = UIView.autoLayoutCapapble()
+		
+		translationLabel.numberOfLines = 0;
+		languageLabel.numberOfLines = 0;
+		contextLabel.numberOfLines = 0;
+		
+		if let nav = self.navigationController {
+			translationLabel.preferredMaxLayoutWidth = nav.view.frame.size.width - 40
+			languageLabel.preferredMaxLayoutWidth = nav.view.frame.size.width - 40
+			contextLabel.preferredMaxLayoutWidth = nav.view.frame.size.width - 40
+		} else {
+			translationLabel.preferredMaxLayoutWidth = self.view.frame.size.width - 40
+			languageLabel.preferredMaxLayoutWidth = self.view.frame.size.width - 40
+			contextLabel.preferredMaxLayoutWidth = self.view.frame.size.width - 40
+		}
+		
+		translationLabel.font = UIFont.boldSystemFontOfSize(20)
+		languageLabel.font = UIFont.systemFontOfSize(14)
+		languageLabel.textColor = UIColor.lightGrayColor()
+		contextLabel.font = UIFont.systemFontOfSize(16)
+		
+		translationLabel.text = String(format: "TRANSLATED_TO_%@%@".localized, arguments: [bookmark.word, bookmark.translation[0]])
```

Comment:
Agreed again 😜, how would you suggest the alternative? Because the `translated to` way was the nicest layout I could come up with so far.